utils/ExampleRunner/template-config.js

```diff
@@ -57,6 +57,9 @@ module.exports = {
   module: {
     rules,
   },
+  experiments: {
```

Comment:
Is this a good idea for production code?  Or does it only affect running the examples?

Reply:
I think this is the new standard in webpack https://webpack.js.org/configuration/experiments/

> asyncWebAssembly: Support the new WebAssembly according to the [updated specification](https://github.com/WebAssembly/esm-integration), it makes a WebAssembly module an async module. And it is enabled by default when [experiments.futureDefaults](https://webpack.js.org/configuration/experiments/#experimentsfuturedefaults) is set to true.